### PR TITLE
Fix/OB-3725 when renaming a hotspot group an unexpected create sequence event is received

### PR DESCRIPTION
**Pull request by MAG-ChristopherAtkinson**
Updated Update and Rename sequence to use the new endpoint, and updated test due to change in expected behaviour.

### CODE REVIEW

File: Library/src/Systems/Sequence/SequenceSystem.cpp [hunk 1]
```diff
@@ -103,7 +103,26 @@ void SequenceSystem::UpdateSequence(const String& SequenceKey,
 									const csp::common::Map<csp::common::String, csp::common::String>& MetaData,
 									SequenceResultCallback Callback)
 {
-	CreateSequence(SequenceKey, ReferenceType, ReferenceId, Items, MetaData, Callback);
+	if (!ValidateKey(SequenceKey))
+	{
+		CSP_LOG_FORMAT(csp::systems::LogLevel::Error,
```
Comment (MAG-OR) [inline Library/src/Systems/Sequence/SequenceSystem.cpp:108]: [Question] why not use CSP_LOG_ERROR_FORMAT?

Comment (MAG-ChristopherAtkinson) [reply Library/src/Systems/Sequence/SequenceSystem.cpp:108]: This was based on the format of the related code, so no particular reason so happy to update for readability.

Comment (MAG-OR) [reply Library/src/Systems/Sequence/SequenceSystem.cpp:108]: nah I was just curious. thanks!

**Review by MAG-ElliotMorris — APPROVED** [review]
LGTM

**Pull request merged**